fixtures/integrations/jira/stub_client.py

```diff
@@ -11,6 +11,18 @@ def get_create_meta_for_project(self, project):
             response["projects"][0]["id"] = "10001"
         return response["projects"][0]
 
+    def get_issue_fields(self, project_id, issue_type_id):
+        return self._get_stub_data("issue_fields_response.json")
+
+    def get_issue_types(self, project_id):
+        return self._get_stub_data("issue_types_response.json")
+
+    def get_priorities(self):
+        return self._get_stub_data("priorities_response.json")
+
+    def get_versions(self, project_id):
```

Comment:
It's a little concerning to me that I just added stubs for getting the versions and priorities, I think the Jira Cloud tests are missing coverage.